Dockerfile

```diff
@@ -27,7 +27,7 @@ ADD LICENSE /usr/share/doc/python-crane/
 ADD COPYRIGHT /usr/share/doc/python-crane/
 ADD README.rst /usr/share/doc/python-crane/
 
-RUN pip-python install /usr/local/src/crane/
+RUN pip install /usr/local/src/crane/
```

Comment:
@bmbouter I needed to updated this in order to build the docker image. Ran the command `docker build --rm=true -t pulp/crane .` on a EL7 system.